src/Stripe/Infrastructure/ParameterBuilder.cs

```diff
@@ -77,6 +77,10 @@ public static string ApplyAllParameters(this StripeService service, object obj,
                             var stripeAccountBankAccountOptions = (StripeAccountBankAccountOptions)value;
                             newUrl = ApplyNestedObjectProperties(newUrl, stripeAccountBankAccountOptions);
                         }
+                        else if (obj.GetType() == typeof(StripeEvidence))
```

Comment:
Fortunately, this isn't necessary anymore. Nested options can just be marked with INestedOption instead and it magically works.